src/main/daml/Daml/Finance/Interface/Types/Date/Calendar.daml

```diff
@@ -5,6 +5,16 @@ module Daml.Finance.Interface.Types.Date.Calendar where
 
 import DA.Date
 
+-- | Key used to look up the holiday calendar of an entity, as defined by a reference data provider.
+data HolidayCalendarKey = HolidayCalendarKey
```

Comment:
I feel this key should be part of Interface.Data

Reply:
@matteolimberto-da that worked, but had move the build of `Daml.Finance.Interface.Claims` (3 steps down in the list) such that it is built after the `Daml.Finance.Interface.Data` package (which it depends on).

Reply:
👍